src/main/resources/assets/nei/cfg/serialhandlers.cfg

```diff
@@ -0,0 +1,6 @@
+# This version of NEI loads recipes in parallel. Some recipe handlers have not been designed with that in mind and might
+# crash. List these handlers in this file to force them to be loaded a single thread. Each line in this file should
+# either be a comment (starts with '#') or a fully qualified class name (that includes the package name) for a recipe
+# handler.
+# If you delete this file, it will be regenerated with the default height hacked handlers list.
```

Comment:
"height hacked handlers" should be "serial handlers"

Reply:
Oops 😄